src/agent/src/image_rpc.rs

```diff
@@ -197,8 +194,58 @@ impl ImageService {
                 return Err(e);
             }
         };
+        self.add_image(String::from(image), String::from(cid)).await;
+        Ok(())
+    }
+
+    /// Pull image when creating container and return the bunld path with rootfs.
+    pub async fn pull_image_in_create_container(&self, image: &str, cid: &str) -> Result<String> {
+        Self::set_proxy_env_vars();
+        if image.contains("pause") {
+            let mount_path = Self::unpack_pause_image(cid, "pause")?;
```

Comment:
Why do we add an additional level in the directory path, such as `pause` and `images`?